fetcher/src/job/job.controller.ts

```diff
@@ -37,6 +41,17 @@ export class JobController {
     return await this.stopFetcher({ aggregatorHash, chain })
   }
 
+  private async activeFetchers() {
+    const activeAggregators = await loadActiveAggregators({
```

Comment:
Here, we can see that we load activate aggregators and then process them resulted in smaller set of them. It would be probably good to rename the function from `activeFetchers` to `activeAggregators`. Or is there some reason to call them as fetchers?